types/office-js-preview/index.d.ts

```diff
@@ -411,6 +411,9 @@ declare namespace Office {
     }
     /**
      * Specifies the host Office application in which the add-in is running.
+     * 
+     * @remarks
+     * [Api set: ExcelApi 1.1, WordApi 1.1, PowerPointApi 1.1, Mailbox 1.5]
```

Comment:
As we discussed, let's not list the Api sets like this since this is available in Common and predate RichApis, for example.